content/java/concepts/strings/terms/equalsIgnoreCase/equalsIgnoreCase.md

```diff
@@ -0,0 +1,35 @@
+---
+Title: '.equalsIgnoreCase()'
+Description: 'This method compares strings.'
+Subjects:
+  - 'Computer Science'
+Tags:
+  - 'Methods'
+  - 'String'
+CatalogContent:
+  - 'learn-java'
+---
+
+The `equalsIgnoreCase` method compares the strings in the program and then gives an output of true or false. It is not a case sensitive method, so this method does not consider this as a factor while comparing the strings.
+
+## Syntax
+
+```pseudo
```

Comment:
This more of an example. Syntax should be more general. Like this:
```
boolean result = stringOne.equalsIgnoreCase(stringTwo);
```
Then have a paragraph after the syntax example explaining what `stringOne` and `stringTwo` are, and how `result` is determined.